script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_2.txt

```diff
@@ -60,7 +60,7 @@ In Japanese terms, you could call it a bit of skinship? Or more like exchanging
 ...I don't think that's the right word for what just happened.
 For a teacher, that kind of prank is way too much―――
 <Page20>
-"...What did you need me for, anyway?
+"...What did you need me for, anyways?
```

Comment:
I believe this should stay `anyway`. Made a similar comment on the group PR I think
https://writingexplained.org/anyway-or-anyways-difference

Reply:
Huh I made the right edit and tried to undo that. Good job me then